internal/cmd/intake/runner/delete/delete.go

```diff
@@ -0,0 +1,105 @@
+package delete
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/params"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	cliErr "github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/intake/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/intake"
+)
+
+const (
+	runnerIdArg = "RUNNER_ID"
+)
+
+// inputModel struct holds all the input parameters for the command
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	RunnerId string
+}
+
+// NewDeleteCmd creates a new cobra command for deleting an Intake Runner
+func NewDeleteCmd(p *params.CmdParams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("delete %s", runnerIdArg),
+		Short: "Deletes an Intake Runner",
+		Long:  "Deletes an Intake Runner.",
+		Args:  args.SingleArg(runnerIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Delete an Intake Runner with ID "xxx"`,
+				`$ stackit intake runner delete xxx`),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p.Printer, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p.Printer)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to delete Intake Runner %q?", model.RunnerId)
+				err = p.Printer.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			if err = req.Execute(); err != nil {
+				return fmt.Errorf("delete Intake Runner: %w", err)
+			}
+
+			p.Printer.Info("Deletion request for Intake Runner %q sent successfully.\n", model.RunnerId)
```

Comment:
```suggestion
			p.Outputf("Deletion request for Intake Runner %q sent successfully.\n", model.RunnerId)
```

Be aware, ´p.Printer.Info()´ goes to stderr, while `p.Printer.Outputf()` goes to stdout. 

https://github.com/stackitcloud/stackit-cli/blob/9dfc1f315fc2921537d93e84e36b36fa01a68bbd/internal/pkg/print/print.go#L65-L73

https://github.com/stackitcloud/stackit-cli/blob/9dfc1f315fc2921537d93e84e36b36fa01a68bbd/internal/pkg/print/print.go#L104-L111